packages/augur-ui/src/modules/modal/transfer-form.tsx

```diff
@@ -92,15 +96,19 @@ export class TransferForm extends Component<
   };
 
   handleMax = () => {
-    const { balances, fallBackGasCosts, GsnEnabled, gasPrice } = this.props;
+    const { balances, useSigner, fallBackGasCosts, GsnEnabled, gasPrice } = this.props;
     const { currency, relayerGasCosts } = this.state;
 
+    const balance = useSigner
+      ? balances[`${currency.toLowerCase()}NonSafe`]
```

Comment:
not a fan of composite string keys, hard to search and find usages when refactoring use the dot notation balances.XXX,